src/editor/components/components/ActionBar/ActionBar.module.scss

```diff
@@ -0,0 +1,22 @@
+@use '../../../style/variables.scss';
+
+.wrapper {
+    position: absolute;
+    background: rgba(55, 55, 55, 0.4);
+    transform: translateX(-50%);
+    bottom: 15%;
+    left: 50%;
+    border: unset;
+    height: 40px;
+    border-radius: 16px;
+    align-items: center;
+    display: flex;
+    column-gap: 8px;
+    padding: 8px 12px;
+    transition: 0.2s ease-in-out;
+    &:hover {
+      background: rgba(55, 55, 55, 0.6);
+    }
+  
+  ;
```

Comment:
Strange semicolon here. Do you have prettier autoformat configured on save in VS code?

Use [vscode](https://code.visualstudio.com/) with the prettier extension for code formatting and configure auto format on save:

- `ctrl+shift+p`, search "install" and select "install extensions", search "prettier", click on "Prettier" and click "Install" button
- Configure format on save go to `File->Preferences->Settings->Text editor->Formatting` and check "Format On Save"
- Open a json/html/js/scss file, `ctrl+shift+p`, search "format", select "Format File", select the prettier formatter

From now on, each time you save, it will auto format with prettier.

Reply:
yep, i have that. for some reason it required a manual run